### PR TITLE
feat: fix build issues and prepare release

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.3] - 2020-03-03
+
+### Fixxed
```
[the remark this answers is not in the document]
Comment (soerenmartius) [reply CHANGELOG.md]: fixed in 20c7cb9

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+- drift types shouldn't be a global type file since it's ignored by tsc
+
+### Fixxed
```
[the remark this answers is not in the document]
Comment (soerenmartius) [reply CHANGELOG.md]: fixed in 20c7cb9

**Comment by soerenmartius** [comment]
> Build stage naming is also bad. we should fix this.

renamed to CI Pipeline

**Comment by mariux** [comment]
> > Build stage naming is also bad. we should fix this.
> 
> renamed to CI Pipeline

the publish pipeline still has bad naming (called build)

**Comment by soerenmartius** [comment]
> > > Build stage naming is also bad. we should fix this.
> > 
> > 
> > renamed to CI Pipeline
> 
> the publish pipeline still has bad naming (called build)

renamed

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   build:
```
Comment (mariux) [inline .github/workflows/publish.yml:8]:
```suggestion
  publish:
```

**Review by mariux — CHANGES_REQUESTED** [review]

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**